client/src/api/Queries.tsx

```diff
@@ -30,22 +31,55 @@ export function useGet<T>(
       })
       .then((data) => data.data)
   );
-  if ((isLoading || isFetching) && queryState.open === false) {
-    setqueryState({
-      open: true,
-    });
-  } else if (isError) {
-    showError(errorMessage);
-  } else if (
-    !isLoading &&
-    !isFetching &&
-    !isError &&
-    queryState.open === true
-  ) {
-    setqueryState({
-      open: false,
-    });
-  }
+
+  // if ((isLoading || isFetching) && queryState.open === false) {
+  //   // TOOD js (25.03.2021): The setQueryState() seems to cause problems like "Cannot update a component (`Batcher`) while rendering a different component"
+  //   // in the console output (see https://github.com/facebookexperimental/Recoil/issues/12#issuecomment-648620138 for an explanation).
+  //   // Recoils seems to be working on a fix
+  //   // Possible fix: https://github.com/facebookexperimental/Recoil/issues/12#issuecomment-732193801
+  //   setQueryState({
+  //     open: true,
+  //   });
+  // } else if (isError) {
+  //   showError(errorMessage);
+  // } else if (
+  //   !isLoading &&
+  //   !isFetching &&
+  //   !isError &&
+  //   queryState.open === true
+  // ) {
+  //   setQueryState({
+  //     open: false,
+  //   });
+  // }
+
+  useEffect(() => {
+    if ((isLoading || isFetching) && queryState.open === false) {
```

Comment:
Das Setzen des _queryState_ (inkl. der dazugehörenden Logik) habe ich hier mal in die _useEffect_ Funktion gezügelt. Das scheint zu helfen.